CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/) 
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [Unreleased]
+
+### Added
+- Add an option to keep the UI always on by setting the hideTimeout to -1
```

Comment:
Please stick to the pattern used in this document so a reader can immediately see which component this config option belongs to: Add an option to keep the UI always visible by setting `UIContainerConfig#hideTimeout` to `-1`